src/Symfony/Component/ErrorHandler/ErrorRenderer/ErrorRendererInterface.php

```diff
@@ -23,5 +23,5 @@ interface ErrorRendererInterface
     /**
      * Renders a Throwable as a FlattenException.
      */
-    public function render(\Throwable $exception): FlattenException;
+    public function render(\Throwable $throwable): FlattenException;
```

Comment:
I had to check to be sure, and the psalm errors confirmed: we don't rename arguments to avoid conflict when merging older branches into the newer. Even if I agree that this name would have been more accurate for a new code.

Reply:
got it, will rename back in a few. Thanks!

Reply:
done